.github/workflows/_Test-OCI-Factory.yaml

```diff
@@ -56,6 +56,9 @@ jobs:
       # test-oci-factory/pytest requirements
       - run: pip install -r tests/etc/requirements.txt
 
+      # build_rock/configure requirements 
+      - run: pip install -r src/build_rock/configure/requirements.txt
```

Comment:
Why not have this in `tests/etc/requirements.txt` too? If you want to avoid duplication it should be possible to just add `-r ../../src/build_rock/configure/requirements.txt` to that file

Reply:
Sure! That is pretty cool.